src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -24,6 +24,7 @@ class DetectorType(Enum):
     DUPLICATE_SPANS = "duplicates"
     SEQUENTIAL_SLOW_SPANS = "sequential"
     LONG_TASK_SPANS = "long_task"
+    RENDER_BLOCKING_ASSET_SPAN = "render_blocking_assets"
```

Comment:
@k-fish Your comment above mentions tag key length limits - do you know what that limit is offhand? This name is probably too long 😬

Reply:
I already counted when I did the review, it should be fine. (limit is 32, that sentence is 22, we add less than 10 elsewhere, it's close but 🤷 )

Reply:
> Tag keys have a maximum length of 32 characters and can contain only letters (a-zA-Z), numbers (0-9), underscores (_), periods (.), colons (:), and dashes (-).
>
> Tag values have a maximum length of 200 characters and they cannot contain the newline (\n) character.